model/flow/execution_result.go

```diff
@@ -14,6 +14,7 @@ type ExecutionResult struct {
 	BlockID          Identifier // commit of the current block
 	Chunks           ChunkList
 	ServiceEvents    ServiceEventList
+	ExecutionDataID  Identifier
```

Comment:
As this will permanently change the shape of `ExecutionResults` and therefore how their IDs are calculated, i wanted to make sure we've properly though through this change, and that it makes sense.

Reply:
Thanks, the change makes sense, but the thing remaining that I need to check is if this will be populated properly for the root block of a spork.